composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "boxuk/qm-memcache-stats",
     "description": "WordPress plugin to provide Memcache Stats to the awesome Query Monitor plugin.",
     "license": "GPLv2",
+    "type": "wordpress-plugin",
```

Comment:
This will put it in the plugins directory. Which may be ok, or should it be an `wordpress-muplugin`?

Reply:
Oh cool, yes it should be, thanks.